lib/model/frames/form.js

```diff
@@ -140,11 +140,16 @@ Form.Partial = class extends Form {};
 
 Form.Extended = class extends Frame.define(
   'submissions',        readable,       'lastSubmission', readable,
+  'received',           readable,       'has_issues',       readable,
+  'edited',             readable,
```

Comment:
Could it be confusing that the `edited` field isn't about the form itself and whether it's been edited, but is instead about the form's submissions? Would `editedSubmissions` be better? (Though I guess that pattern doesn't really work for the `hasIssues` review state...) Another alternative would be to have a single field (named `reviewStates`?) that is an object with a property for each of these review states. Quite possibly no change needed, mostly just wanted to flag a potential for confusion.

Reply:
I changed it a bit so the states on the `Form` frame are still flat, but they get put into a `reviewState`(s - ahh maybe it should be pluralized) object for the API response.

Reply:
Ah nice! I do think it's helpful to group them together like that. In that case, I think these fields shouldn't be `readable`, since they won't be available as top-level fields on the API response.

Another strategy that might be possible would be to define a frame for the `reviewState` object, then use `extender()` to insert these counts into that frame. (I'm not quite sure about the details, but I think there are similar examples in the codebase.) Possibly it doesn't add much here though, since you already have to have a custom `forApi()` in order for `submissions` to default to 0.